### PR TITLE
 Regenerate testing_v1 Java API

### PR DESCRIPTION
**Pull request by bootstraponline**
Remove Java cloud_api_poc

Update API JSON

### CODE REVIEW

File: gradle/gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -2,4 +2,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.3.1-all.zip
```
Comment (runningcode) [inline gradle/gradle/wrapper/gradle-wrapper.properties]: don't forget to check in the update gradle.jar file.

Comment (bootstraponline) [reply gradle/gradle/wrapper/gradle-wrapper.properties]: you mean gradle-wrapper.jar? that one hasn't changed

Comment (runningcode) [reply gradle/gradle/wrapper/gradle-wrapper.properties]: yeah `gradle-wrapper.jar` it definitely changes with each version.

Comment (bootstraponline) [reply gradle/gradle/wrapper/gradle-wrapper.properties]: I edited the distributionUrl directly which is why there's no change. maybe the gradle wrapper command updates that jar

Comment (bootstraponline) [reply gradle/gradle/wrapper/gradle-wrapper.properties]: updated.

**Comment by renas** [comment]
Thanks @bootstraponline

**Pull request merged**